amz_review_scraper/asin_validation.py

```diff
@@ -4,10 +4,9 @@
 # example of seemingly valid ASIN = B011111111
 
 
-
 def get_valid_asin():
     while True:
-        asin = input("Please enter a vaild ASIN: ")
+        asin = input("Please enter a vaild ASIN: ").strip().upper()
```

Comment:
nice, I could check, but this is more about how to think, if you removed these would all the tests still pass?